lib/oedipus_lex.rb

```diff
@@ -44,14 +44,7 @@ def to_ruby state, predicates, exclusive
         start_state == state or
         (state.nil? and predicates.include? start_state)
 
-      cond =
-        if exclusive or not start_state then
-          "when text = ss.scan(#{regexp}) then"
-        elsif start_state =~ /^:/ then
-          "when (state == #{start_state}) && (text = ss.scan(#{regexp})) then"
-        else
-          "when #{start_state} && (text = ss.scan(#{regexp})) then"
-        end
```

Comment:
Well... brakeman + your corpus is a good exercise for ruby_parser, but that really doesn't exercise oedipus_lex because its only one lexer specification.

That said, if you think they're covered, then they probably are. Should be pretty easy to tell. I've merged this PR w/ my changes from above.